uiautomationutilities/build.gradle

```diff
@@ -105,6 +106,7 @@ android {
         buildConfigField("String", "UPDATE_SOURCE_LOCAL_APK", "\"$appSourceLocalApk\"")
         buildConfigField("boolean", "PREFER_PRE_INSTALLED_APKS", "$usePreInstalledApks")
         buildConfigField("boolean", "PRE_INSTALL_LTW", "$preInstallLtw")
+        buildConfigField("boolean", "SEND_POWERLIFT_FAILURES", "$sendPowerLiftLogs")
```

Comment:
Perhaps rename the field to `SEND_POWERLIFT_LOGS`

Reply:
Sounds good, I'm not sure why I named it that honestly, considering the build param is already named with "logs"